internal/tls/config_test.go

```diff
@@ -111,6 +105,20 @@ func Test_tlsClientConfigFromSecret(t *testing.T) {
 	}
 }
 
+func TestKubeTLSConfigFromSecret(t *testing.T) {
```

Comment:
If you want to go above and beyond, I'd suggest to add a test for `TLSClientConfigFromSecret` to make sure it doesn't fail as it did before.